config/jobs/kubernetes/sig-node/OWNERS

```diff
@@ -1,27 +1,9 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-  - bart0sh
-  - bobbypage
-  - derekwaynecarr
-  - endocrimes
-  - pacoxu
-  - sjenning
-  - SergeyKanzhelev
-  - kannon92
+  - sig-node-test-reviewers
 approvers:
-  - yujuhong
-  - Random-Liu
-  - dchen1107
-  - derekwaynecarr
-  - sjenning
-  - mrunalp
-  - klueska
-  - SergeyKanzhelev
-  - endocrimes
-  - kannon92
-emeritus_approvers:
```

Comment:
The emeritus list shouldn't be deleted?

Reply:
I moved emeritus to OWNER_ALIASES.

Reply:
I see, that does obscure the history of _which_ packages they owned at the time though ...